dotCMS/src/main/java/com/dotmarketing/servlets/BinaryExporterServlet.java

```diff
@@ -69,9 +68,10 @@
  *
  */
 public class BinaryExporterServlet extends HttpServlet {
-
+    
+    private boolean isContentLive = false;
+    private boolean isContentExpired = false;

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this misleading mutable servlet instance field or make it "static" and/or "final" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2226)